### DISCUSSION TITLE
Map is having high index and elements are going back

### DISCUSSION THREAD

**Discussion by Shesharaopuli** [capacitor-community/google-maps#108] [General]
I have integrated this plugin but it is showing on top of all elements.

As I cannot run in browser this plugin I cannot do try and error. 

Could you please help?

**Comment by hemangsk** [comment]
This problem of map showing on top of all elements is fixed by Tafel on their fork at https://github.com/DutchConcepts/capacitor-google-maps/tree/next . Its not on npm as of now so you'll have to install that plugin from GitHub https://github.com/DutchConcepts/capacitor-google-maps/releases/tag/v2.0.0-alpha.4

**Reply by Shesharaopuli** [reply]
I have just installed this.
But I am unable to see  reverseGeocodeCoordinate function.

And coming to your plugin, except that fixed element issue, your plugin is simply awesome!!

I used to take address by sending coodinates to reverseGeocodeCoordinate()  and pass that address as title to the create() function.

How to use reverseGeocodeCoordinate() in Dutch concepts plugin ?

**Reply by Shesharaopuli** [reply]
Could you please reply Hemang?

**Reply by tafelnl** [reply]
We've just released a (pretty stable) beta for version 2 of this plugin under the name [`@capacitor-community/google-maps`](https://www.npmjs.com/package/@capacitor-community/google-maps).

It features a more complete and consistent API. Also the map can finally be rendered behind the webview in this new version.

Extensive documentation for v2 can be found here: https://capacitor-community.github.io/google-maps/

Since version 1 lacked a lot of (basic) features, did not have good coding practices, had a poor API and it is not recommended to use it anymore, we've dropped support for it. Therefore I will close this issue. We believe version 2 offers a better Developer Experience and is a much better fit for anyones use case.
